blog.html

```diff
@@ -4,12 +4,310 @@
+                            <time datetime="2022-06-20" class="caption-date">20 JUN 2022</time>&#10072;
+                            <p class="caption-author">TONY NGUYEN</p>
+                            <p class="tag-text">GAMING</p>
+                            <a href="blog-article.html">READ MORE &rarr;</a>
+                        </figcaption>
+                    </figure>
+                </li>
+                <li class="list-item">
+                    <figure class="item-wrap">
+                        <img src="#" alt="t-rex" class="item-image">
+                        <img src="#" alt="human icon" class="item-logo">
+                        <img src="#" alt="calendar icon" class="item-logo">
+                        <img src="#" alt="gaming tag" class="item-tag">
+                        <figcaption class="list-caption">
+                            <h2 class="caption-heading">SONIC LIVE</h2>
+                            <p class="caption-description">On September 22, 2021. A volutpat nulla eros, nulla ultrices
+                                nec nulla imperdiet...</p>
+                            <!-- I understand this separator {&#10072;} can be styled with css instead of symbol -->
+                            <time datetime="2022-06-20" class="caption-date">20 JUN 2022</time>&#10072;
+                            <p class="caption-author">TONY NGUYEN</p>
+                            <p class="tag-text">NFT</p>
+                            <a href="blog-article.html">READ MORE &rarr;</a>
+                        </figcaption>
+                    </figure>
+                </li>
+                <li class="list-item">
+                    <figure class="item-wrap">
+                        <img src="#" alt="duck" class="item-image">
+                        <img src="#" alt="human icon" class="item-logo">
+                        <img src="#" alt="calendar icon" class="item-logo">
+                        <img src="#" alt="gaming tag" class="item-tag">
+                        <figcaption class="list-caption">
+                            <h2 class="caption-heading">SONIC LIVE</h2>
+                            <p class="caption-description">On September 22, 2021. A volutpat nulla eros, nulla ultrices
+                                nec nulla imperdiet...</p>
+                            <!-- I understand this separator {&#10072;} can be styled with css instead of symbol -->
+                            <time datetime="2022-06-20" class="caption-date">20 JUN 2022</time>&#10072;
+                            <p class="caption-author">TONY NGUYEN</p>
+                            <p class="tag-text">NFT</p>
+                            <a href="blog-article.html">READ MORE &rarr;</a>
+                        </figcaption>
+                    </figure>
+                </li>
+            </ul>
+        </section>
+        <div class="next-previous-buttons">

```

Comment:
Commonly, navigation elements for moving between pages are named "pagination". Using standard terminology helps maintain consistency and clarity.